gnssanalysis/_version.py

```diff
@@ -70,13 +77,19 @@ def decorate(f):
     return decorate
 
 
-def run_command(commands, args, cwd=None, verbose=False, hide_stderr=False,
-                env=None):
+def run_command(
+    commands: List[str],
+    args: List[str],
+    cwd: Optional[str] = None,
+    verbose: bool = False,
+    hide_stderr: bool = False,
+    env: Optional[Dict[str, str]] = None,
+) -> Tuple[Optional[str], Optional[int]]:
     """Call the given command(s)."""
```

Comment:
One-line sphinx docstring (missing the description of input parameters, errors raised, what's returned, etc.: `:param` `:return`)